src/web/app/pages-instructor/instructor-search-page/instructor-search-page.component.ts

```diff
@@ -84,4 +106,85 @@ export class InstructorSearchPageComponent implements OnInit {
       this.statusMessageService.showErrorToast(resp.error.message);
     });
   }
+
+  private getSearchCommentsTable(searchResults: CommentSearchResult[]): SearchCommentsTable[] {
+    return searchResults.map((res: CommentSearchResult) => ({
+      feedbackSession: res.feedbackSession,
+      questions: res.questions,
+    }));
+  }
+
+  private getCoursesWithStudents(students: Student[]): SearchStudentsListRowTable[] {
+    const distinctCourses: string[] = Array.from(
+        new Set(students.map((s: Student) => s.courseId)),
+    );
+    const coursesWithStudents: SearchStudentsListRowTable[] = distinctCourses.map(
+        (courseId: string) => ({
+          courseId,
+          students: Array.from(
+              new Set(
+                  students
+                      .filter((s: Student) => s.courseId === courseId),
+              ),
+          ).map((s: Student) => ({
```

Comment:
👍 . Yup this is what we were looking for with the refactoring!

Reply:
Got it. Could I just remove the tests for the functions removed? Or do I port them over to the components that are using them?

Reply:
If you could port them over in this PR, it would be great